tasks/install.yaml

```
@@ -0,0 +1,31 @@
+- block:
+
+  - name: Add MariaDB repository key
+    apt_key:
+      url: "{{ mariadb.repository.apt.key_url }}"
+
+  - name: Add MariaDB repository
+    apt_repository:
+      repo: "{{ mariadb.repository.apt.repository }}"
+      filename: /etc/apt/sources.list.d/mariadb.list
+
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Install MariaDB
+  when:
+    - ansible_distribution == 'Ubuntu'
+    - mariadb.galera.arbitrator == False
```

This task won't work on a proServer so we cannot use an or-condition here.
```suggestion
  when: ansible_distribution == 'Ubuntu' and not mariadb.galera.arbitrator
```

---

This is not an or-condition; it is an and-condition https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#the-when-statement

---

Ah, I remembered wrong :+1:

---

boolean access changed in 560e3e00863143a37b458ba405c69824f92ca888